standards_lab/settings/settings.py

```diff
@@ -166,6 +166,7 @@
 EDIT_MODE = True
 
 ROOT_PROJECTS_DIR = env("ROOT_PROJECTS_DIR")
+os.makedirs(ROOT_PROJECTS_DIR, exist_ok=True)
 
```

Comment:
Hmm I toyed with doing this... but I think it is a bit risky to have in the default settings file. We effectively blindly serve the contents of the directory, if someone had multiple instances or there was other non standards-lab stuff in the location things could go wrong. Could you do the mkdir in the setup?

Reply:
Maybe you could do it in the docker file? just a mkdir -p in CMD

Reply:
Yes, I can move it there. I put it in settings.py because Rob also ran into trouble with this locally, and the error message was quite opaque. Maybe we could just raise a better error.

Reply:
Done.